src/App/Zinc.DeveloperCenter.Client.Web/projects/zn-developercenter/src/app/screens/adr-list/components/repo-summary/repo-list.component.ts

```diff
@@ -42,4 +45,43 @@ export class RepoListComponent implements OnInit, OnDestroy {
                 this.fetchedRepos = true;
             });
     }
+
+    // sort functions will sort Adrs on one of three options,
+    // or they will swap asc/desc.
+
+    public sortByLastUpdatedDate(): void {
+        if (this.sortedOn === 'lud')
+        {
```

Comment:
Codacy has a fix for the issue: [Opening curly brace does not appear on the same line as controlling statement.](https://app.codacy.com/gh/GSFSGroup/Zinc.DeveloperCenter/pullRequest?prid=9805944)
```suggestion
            this.sortAsc = !this.sortAsc;
```